content/en/Getting started/Assets/_index.md

```diff
@@ -13,9 +13,8 @@ Help our pentesters test your [assets](../glossary#asset) faster.
 The **Let's get started!** screen includes two options:
 
 - **Create a new pentest from an existing asset**
-  - This option opens a drop-down text box. Use it to select from assets that you've
-    created. It populates the Asset screen with available information.
-- **Create a new pentest for a new asset**.
+  - From the list of your assets that appears, select an asset to proceed with. Once you click **Next**, you can see the Asset screen with available information.
```

Comment:
I appreciate the change to a more "active voice". I have a couple of other suggestions, including our guidance to "Select" buttons, ref https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click

```suggestion
  - From the list of your assets that appears, choose an asset. Once you select **Next**, you can see the Asset screen with available information.
```